index.bs

```diff
@@ -128,7 +123,7 @@ spec: webidl
       or other sources this specification hasn't anticipated.
     </dd>
 
-    <dt><dfn export>Powerful feature</dfn></dt>
+    <dt><dfn export lt="powerful feature|feature">Powerful feature</dfn></dt>
```

Comment:
Missed the possibility of `local-lt` here too: Probably we don't want this spec to define "feature" for the whole world?